src/components/Proposals/ProposalDetail.js

```diff
@@ -17,12 +17,47 @@ import {
 } from '@aragon/ui'
 import { safeDiv } from '../../lib/math-utils'
 import { useGetVote } from '../../hooks/disputable-voting-logic'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_DISPUTED,
+  VOTE_STATUS_PAUSED,
+} from './disputable-vote-statuses'
 import DisputableActionStatus from './DisputableActionStatus'
 import InfoBoxes from './InfoBoxes'
 import SummaryBar from './SummaryBar'
+import FeedbackModule from './FeedbackModule'
 import Layout from '../Layout'
 import { networkEnvironment } from '../../current-environment'
 
+function getAttributes(status, theme) {
+  const attributes = {
+    [VOTE_STATUS_CANCELLED]: {
+      background: '#F9FAFC',
+      border: '#DDE4E9',
```

Comment:
If these colours are not available in the base theme perhaps we should extend our own theme object to avoid repeating them in multiple places throughout the app?

Reply:
good idea!

Reply:
I think it would be good to update our existing palette with the new values if they are different in the design. If you look at the [colours page on devbox](https://devbox.dweb.cool/#Colors) we have a lot of crossover between the various `warning` `positive` `negative`, we can update them with new values in our [theme](https://github.com/aragon/network-dashboard/blob/master/src/theme.js) and access them with `useTheme`.

For example the `challenged` colours are very close to `warning`, we could just assign them to `warning` in our theme and use them from there, rather than having custom hex values in the templates themselves.